packages/docusaurus-theme-classic/codeTranslations/base.json

```diff
@@ -61,8 +61,8 @@
   "theme.docs.versions.unreleasedVersionLabel": "This is unreleased documentation for {siteTitle} {versionLabel} version.",
   "theme.lastUpdated.atDate": "on {date}",
   "theme.lastUpdated.byUser": "by {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "Last updated{atDate}{byUser}",
+  "theme.lastUpdated.lastUpdatedAtBy": "Last updated {atDate} {byUser}",
```

Comment:
the missing space was on purpose sorry :)

Reply:
with your implementation, if there's a "byUser" but no "atDate", the string will contain a double space: `Last updated  by Sébastien` => not good

Reply:
https://github.com/facebook/docusaurus/pull/4303/files#diff-6d59037256651db4dbb0c9b16ed5c10dc97a8de3cb6311b479266b6816047b48L73

https://github.com/facebook/docusaurus/pull/4303/files#diff-6d59037256651db4dbb0c9b16ed5c10dc97a8de3cb6311b479266b6816047b48L73

Reply:
But they are present on purpose, in case one of the 2 blocks is absent. We can't hardcode the spaces directly in the translation string for that specific reason

Reply:
https://github.com/facebook/docusaurus/compare/9f348e79df854929d1550f7b9c188c9162f8ef15..9a347bcfd9d2fca8bfeb6e7e94dbc0066279d689

Changed where spaces are inserted.

Reply:
thanks, that make sense to me

So what I understand is that the japan sentence for this has no space at all right?

![image](https://user-images.githubusercontent.com/749374/109535323-b986ed80-7abc-11eb-9165-5b860c6f237d.png)

Reply:
thanks, didn't know :)